agent/agent_test.go

```diff
@@ -714,7 +714,7 @@ func TestAgent_UnixRemoteForwarding(t *testing.T) {
 		var err error
 		conn, err = net.Dial("unix", remoteSocketPath)
 		return err == nil
-	}, testutil.WaitLong, testutil.IntervalFast)
+	}, testutil.WaitShort, testutil.IntervalFast)
```

Comment:
This change is not in sync with note in the comments. Do we actually want to do this? Perhaps a separate PR that can then be reverted at some point? Or maybe we'll end up keeping this but if we can't figure out the problem we should revert until such a time that we do.

Reply:
You're right.  I forgot that I changed this in this branch while I was hunting for a repro of our test failures.  I'll remove it and we can revert to `Short` in another PR that isn't about diagnostics like this one is.